### PR TITLE
ceilometer: Remove mongodb suport

### PR DESCRIPTION
**Pull request by jsuchome**
Our mongodb version is not properly maintained and has vulnerabilities which are not going to be fixed.

Requires https://build.suse.de/request/show/155990 (already merged)

### CODE REVIEW

**Comment by aplanas** [comment]
can we try 3.6 before dropping mongodb for soc8? or is this the best option?

**Comment by jsuchome** [comment]
I guess we can, but do you have time for that?

**Comment by jsuchome** [comment]
[Drop mongodb support from ceilometer barclamp](https://trello.com/c/SlnFclzY/709-drop-mongodb-support-from-ceilometer-barclamp)

**Comment by jsuchome** [comment]
Testbuild is failing because of 'technology preview' patch that is part of crowbar-openstack package

**Comment by jsuchome** [comment]
@vuntz another review, please?

File: crowbar_framework/app/models/ceilometer_service.rb [hunk 1]
```diff
@@ -243,20 +193,6 @@ def mongodb_ha(new_members, role)
   # primary.
   def validate_minimum_three_nodes_in_cluster(proposal)
```
Comment (vuntz) [inline crowbar_framework/app/models/ceilometer_service.rb]: I think you should simply drop that method as it's doing nothing anymore (so no need to call it anymore).

Comment (jsuchome) [reply crowbar_framework/app/models/ceilometer_service.rb]: Oh, true.

**Pull request merged**